CSS/style.css

```
@@ -0,0 +1,455 @@
+}
+
+/*hotel search*/
+
+
+.hotel_search {
+	line-height: 24px;
+	text-align: center;
+	color: var(--basic-extra-dark);
+}
+
+.hotel_search h2{
+	font-size: 30px;
+	line-height: 24px;
+	text-align: center;
+	font-weight: bold;
+	color: var(--basic-dark);
+}
+
+.button_hotel_search {
+	width: 568px;
+	height: 86px;
+	background-color: var(--basic-brown);
+	line-height: 26px;
+	font-size: 21px;
+	text-align: center;
+	font-weight: bold;
+	text-transform: uppercase;
+	color:  var(--basic-white);
+}
+
+.button_hotel_search:hover {
+	background-color: var(--basic-medium-brown);
+}
+
+.button_hotel_search:active {
+	background-color: var(--basic-hard-brown);
+}
+
+.date_form {
+	font-size: 14px;
+	line-height: 26px;
+	font-weight: 700;
+	color: var(--basic-dark);
+
+
+}
+
+.arrival , .departure , .button_how_much , .how_much_people{
+	background-color: var(--basic-extra-light-grey);
+	border: none;
+	text-transform: uppercase;
+	font-weight: bold;
+}
+
+.arrival:hover , .departure:hover {
+	background-color: var(--basic-grey-medium);
+}
+
+.arrival:focus , .departure:focus {
+	background-color: var(--basic-white);
+}
+
+.button_how_much , .calendar {		
+									/*Не понимаю как сделать переход. Ниже, в классе .html_logo
+									заменил в background-image: , но не уверен, что это верно.*/
+
+}
+
+.button_find {
+	width: 458px;
+	height: 58px;
+	background-color: var(--basic-blue);
+	color:  var(--basic-white);
+	font-size: 21px;
+	line-height: 26px;
+	text-transform: uppercase;
+	font-weight: bold;
+	border: none;
+}
+
+.button_find:hover{
+	background-color:var(--basic-middle-blue);
+}
+
+.button_find:active{
+	background-color: var(--basic-hard-blue);
+}
+
+
+/*footer*/
+
+
+.main_footer a{
+	font-size: 21px;
+	font-weight: bold;
+	line-height: 26px;
+	color: var(--basic-dark);
+	
+}
+
+.footer_copiright p {
+	line-height: 26px;
+}
+
+.social_button {
+	display: block;
+	width: 46px;
+	height: 48px;
+	background-color: var(--basic-blue);
+	position: relative;
+	text-align: center;	
+}
+
+.youtube_logo , .twitter_logo , .facebook_logo{
+	position: absolute;
+  	top: 50%;
+  	left: 50%;
+  	transform: translate(-50%, -50%);
+}
+
+
+
+.social_button:hover{ 
+	background-color: var(--basic-middle-blue);
+	
+}
+
+.social_button:focus{
+	background-color: var(--basic-hard-blue);
+}
+
+/*Cсылкой стала не только надпись, а вся строка, как исправить или другое решение я пока не придумал
+Вычитал, что img заменить на img можно с помощью JS. Оставить в таком виде, или вернуть прежний?*/
+
+.html_logo {
+	background-image:url("../img/logo_htmlacademy.svg"); 
+	width: 115px;
+	height: 33px;
+}
+
+.html_logo:hover {
+	background-image:url("../img/logo_htmlacademy2.svg"); 
+	width: 115px;
+	height: 33px;
+}
+
+.html_logo:active {
+	background-image:url("../img/logo_htmlacademy3.svg"); 
+	width: 115px;
+	height: 33px;
+}
+
+/*Catalog*/
+
+.selection_of_conditions {
+	background-image: url(../img/background_photo@2x.jpg);
```

```suggestion
	background-image: url("../img/background_photo@2x.jpg");
```
Если в `url` используете кавычки, то нужно использовать их везде.